### PR TITLE
Update the GS guide for React chat components

### PR DESCRIPTION
**Pull request by kazydek**
**Description**
Update the GS guide for React chat components with ideas for improvements from the GS spike.

**Related issue**
[GROW-1345](https://pubnub.atlassian.net/browse/GROW-1345)

### CODE REVIEW

File: lib/docs/stories/introduction/usage.stories.mdx [hunk 1]
```diff
@@ -1,42 +1,63 @@
...
+
+Let's get started!
+
+### Create a PubNub account
+
+Before you start, you need to get **Publish Key** and **Publish Key** for your chat app.
+Sign in or create an account on the [Admin Portal](https://dashboard.pubnub.com/) to create an app and a new keyset. 
 
-## Set up PubNub account
+>**TIP**: When you create a new app on the Admin Portal, the first set of keys is generated automatically, but a single app can have as many keysets as you like. We recommend that you create separate keysets for production and test environments. There are some of the functionalities you might need to enable on your keyset depending on the use case, but they won't be required to complete this Getting Started guide. These options include:
```
Comment (Salet) [inline lib/docs/stories/introduction/usage.stories.mdx:30]: From what I tested, when you create a new PubNub account and select "Chat" as your use case, most of those options will already be enabled. Perhaps we should mention that here?

File: lib/docs/stories/introduction/usage.stories.mdx [hunk 2]
```diff
@@ -1,42 +1,63 @@
...
 - [PubNub React SDK v2.1.0+](https://www.pubnub.com/docs/chat/react/setup)
+- Code editor of your choice (e.g. [Visual Studio Code](https://code.visualstudio.com/download))
+
+## Steps
+
+Let's get started!
+
+### Create a PubNub account
+
+Before you start, you need to get **Publish Key** and **Publish Key** for your chat app.
```
Comment (Salet) [inline lib/docs/stories/introduction/usage.stories.mdx:27]: Publish Key and Publish Key

File: lib/docs/stories/introduction/usage.stories.mdx [hunk 3]
```diff
@@ -1,42 +1,63 @@
...
 
-1. Sign in or create an account to create an app on the
-   [Admin Portal](https://dashboard.pubnub.com/) and get the keys to use in your application.
+### Create a new project for your app
 
-2. When you create a new app, the first set of keys is generated automatically, but a single app can
-   have as many keysets as you like. We recommend that you create separate keysets for production
-   and test environments.
+Create a sample React project you will use as a placeholder for your chat app code. 
+To use the [`react-sample-chat`](https://www.pubnub.com/docs/sdks/javascript/react/#download-the-react-wrapper) app, follow these steps:
```
Comment (Salet) [inline lib/docs/stories/introduction/usage.stories.mdx:38]: Does it make sense to start with an app that already has a working Chat? Perhaps we should start with an empty app? I can provide instructions for that

File: lib/docs/stories/introduction/usage.stories.mdx [hunk 4]
```diff
@@ -1,42 +1,63 @@
...
+```js
+npx create-react-app react-sample-chat
+cd react-sample-chat
+```
 
-## Install and use components
+### Install components
 
-1. Install the components and all required dependencies using npm.
+Add the PubNub JavaScript SDK, React framework packages, and PubNub Chat Components for React:
```
Comment (Salet) [inline lib/docs/stories/introduction/usage.stories.mdx:47]: I would switch "React framework packages" to "PubNub React SDK", React framework itself is already pre-installed when using `create-react-app`

File: lib/docs/stories/introduction/usage.stories.mdx [hunk 5]
```diff
@@ -1,42 +1,63 @@
 import { Meta } from "@storybook/addon-docs/blocks";
 
-<Meta title="Introduction/Usage" />
+<Meta title="Introduction/Getting Started" />
 
-# Usage
+# Getting Started
```
Comment (Salet) [inline lib/docs/stories/introduction/usage.stories.mdx:5]: We should probably mimic all these changes in README.md which is shown on GitHub as well as on npmjs.org

File: lib/docs/stories/introduction/usage.stories.mdx (hunk 2 above)
Comment (kazydek) [reply lib/docs/stories/introduction/usage.stories.mdx:27]: Good catch ;)

File: lib/docs/stories/introduction/usage.stories.mdx (hunk 3 above)
Comment (Salet) [reply lib/docs/stories/introduction/usage.stories.mdx]: Improved that point

**Comment by kazydek** [comment]
Closing the PR as we decided with the team we want to fully revamp them and until that time, we won't make any short-term fixes to the GS guides.

**Pull request closed without merging**